CMakeLists.txt

```diff
@@ -2713,9 +2714,15 @@ target_link_libraries(zig LINK_PUBLIC
     ${LLVM_LIBRARIES}
     ${CMAKE_THREAD_LIBS_INIT}
 )
+
 if(NOT MSVC)
     target_link_libraries(zig LINK_PUBLIC ${LIBXML2})
 endif()
+
+if(MINGW)
+    target_link_libraries(zig LINK_PUBLIC ${Z3_LIBRARIES})
```

Comment:
Can you provide a little background about why this is now necessary? What changed?

Reply:
i don't know if it's just my system or something else, but without this i get linker errors about undefined symbols from z3. i'd like to check on another system though.

Reply:
I'm pretty sure z3 is an optional dependency of LLVM. Are you using the mingw package of llvm or a build from source?

Reply:
i checked on another computer and it seems that this is needed because commit a77a17e2c2dc8c2add8a3ed47708c90799425c70 added the clang static analyzer which has a dependency on libz3.

Reply:
Are you using the mingw clang/llvm package or a source build?

Reply:
the package